src/serverless-functions/send-to-phone.ts

```diff
@@ -0,0 +1,43 @@
+import "source-map-support/register";
+
+import { EFNYC_HOST, serverlessRollbarHandler } from "../utils/serverless-util";
+import { Twilio } from "twilio";
+
+export const handler = serverlessRollbarHandler(async (event) => {
+  const {
+    TWILIO_ACCOUNT_SID,
+    TWILIO_AUTH_TOKEN,
+    TWILIO_PHONE_NUMBER,
+  } = process.env;
+  if (!(TWILIO_ACCOUNT_SID && TWILIO_AUTH_TOKEN && TWILIO_PHONE_NUMBER)) {
+    throw new Error("Twilio credentials are not in environment!");
+  }
+
+  const userPhone =
+    event.queryStringParameters && event.queryStringParameters.userPhone;
+
+  const resultsPagePath =
+    event.queryStringParameters && event.queryStringParameters.userPath;
```

Comment:
It might be good to use `event.body` here instead of `event.queryStringParameters`, making this method something that's used with a POST request instead of a GET, since the function effectively has side effects (in that it sends an SMS message to someone).  You'd then want to change the `sendSMS` function to issue a POST request with the arguments in the body of the request too.

Reply:
Cool cool